app/views/layouts/_header.html.erb

```diff
@@ -5,7 +5,23 @@
       <ul class="nav navbar-nav navbar-right">
         <li><%= link_to t("home"), root_path %></li>
         <li><%= link_to t("help"), help_path %></li>
-        <li><%= link_to t("login"), "##" %></li>
+        <% if logged_in? %>
```

Comment:
Theo e tìm hiểu thì <%=   %> thực thi code và có in ra màn hình còn <% %> thì không có in ra màn hình ạ